luaext/lposix.c

```diff
@@ -330,26 +330,64 @@ static int Pmkfifo(lua_State *L)		/** mkfifo(path) */
 }
 
 
+static void set_cloexec(int fd)
+{
+	int flag = fcntl(fd, F_GETFD);
```

Comment:
Change "flag" to "flags". Even if FD_CLOEXEC is currently the only defined flag, we may as well make the code forward compatible,